packages/cli/src/controllers/telemetry.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { GlobalConfig } from '@n8n/config';
+import { AuthenticatedRequest } from '@n8n/db';
+import { Get, Post, RestController } from '@n8n/decorators';
+import { NextFunction, Response } from 'express';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
+
+@RestController('/telemetry')
+export class TelemetryController {
+	proxy;
+
+	constructor(private readonly globalConfig: GlobalConfig) {
+		this.proxy = createProxyMiddleware({
+			target: this.globalConfig.diagnostics.frontendConfig.split(';')[1],
+			changeOrigin: true,
+			pathRewrite: {
+				'^/proxy/': '/', // /proxy/v1/track -> /v1/track
+			},
+			on: {
+				proxyReq: (proxyReq, req) => {
+					proxyReq.removeHeader('cookie');
```

Comment:
The IP of the user agent might also be contained in the body. Can we strip it out from there, too?

Reply:
You mean the IP we send in the context within the body? There is already a mask in the FE for the IP [here](https://github.com/n8n-io/n8n/blob/36b410abdb16ff0396d42e34fe627a5a3b3eee1f/packages/frontend/editor-ui/src/plugins/telemetry/index.ts#L135
). Also, We can also add a DTO where we can make sure that no value in any of the properties is an IP.

Reply:
After some digging will do a simply DTO, else I would have to transverse each property recursively and for each string check its ipv4 or v6, and the payloads are quite big. Since our FE controls what we send we should be safe in terms of not sending any IP, as we tell the SDK in the FE not to.

Reply:
I double checked to see if there is any IP information, and there is none. 
And I do like the idea to document the payload. I think the best way is to add a section to our privacy page explaining the changes we made and also adding a sample payload.
Did you have anything else in mind @csuermann ?